specification/README.md

```diff
@@ -48,8 +48,8 @@ Configurations include:
   - screenHeight? `number`. The optional screen height in pixels (minimum 0, maximum 10000000).
   - positionX? `number`. Optional override browser X position on screen in pixels (minimum 0, maximum 10000000).
   - positionY? `number`. Optional override browser Y position on screen in pixels (minimum 0, maximum 10000000).
-- upstreamProxyUrl `string`. A socks5 or http proxy url (and optional auth) to use for all HTTP requests in this session. The optional "auth" should be included in the UserInfo section of the url, eg: `http://username:password@proxy.com:80`.
-- upstreamProxyIpMask `object`. Optional settings to mask the Public IP Address of a host machine when using a proxy. This is used by the default BrowserEmulator to mask WebRTC IPs.
+  - upstreamProxyUrl `string`. A socks5 or http proxy url (and optional auth) to use for all HTTP requests in this session. The optional "auth" should be included in the UserInfo section of the url, eg: `http://username:password@proxy.com:80`.
+  - upstreamProxyIpMask `object`. Optional settings to mask the Public IP Address of a host machine when using a proxy. This is used by the default BrowserEmulator to mask WebRTC IPs.
```

Comment:
for got to document the `upstreamProxyUseLocalDns` prop here?

Reply:
Good catch